docs/source/getting-started/content/assembly_getting-started.adoc

```diff
@@ -3,7 +3,9 @@
 
 include::con_understanding-codeready-containers.adoc[leveloffset=+1]
 
-include::ref_minimum-system-requirements.adoc[leveloffset=+1]
+include::proc_installing-codeready-containers.adoc[leveloffset=+1]
+
+include::ref_required-software-packages.adoc[leveloffset=+1]
```

Comment:
Why this order? It would seem more natural to me to swap them. At least on Ubuntu, it seems you need to install libvirt before doing the other steps.